shared/src/akri/configuration.rs

```diff
@@ -4,24 +4,22 @@
 // in favor of camelCase)
 //
 #![allow(non_camel_case_types)]
-
-use super::API_CONFIGURATIONS;
-use super::API_NAMESPACE;
-use super::API_VERSION;
 use k8s_openapi::api::core::v1::PodSpec;
 use k8s_openapi::api::core::v1::ServiceSpec;
+use kube::CustomResource;
 use kube::{
-    api::{ListParams, Object, ObjectList, RawApi, Void},
-    client::APIClient,
+    api::{Api, ListParams, ObjectList},
+    client::Client,
 };
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-pub type KubeAkriConfig = Object<Configuration, Void>;
-pub type KubeAkriConfigList = ObjectList<Object<Configuration, Void>>;
+pub type KubeAkriConfigList = ObjectList<Configuration>;
```

Comment:
should this become ConfigurationList?  (it seems like KubeAkriConfig became Configuration)

Reply:
Good point. Fixed in 67595e71132f